client/constants/validation.ts

```diff
@@ -4,6 +4,36 @@ const minValidationText = 'Must be greater than or equal to ';
 const maxValidationText = 'Must be less than or equal to ';
 
 export const APPLICATION_REVIEW_VALIDATION_SCHEMA = Yup.object().shape({
+  // network application review validation
+  reasonableCostforCommunitySize: Yup.number()
+    .min(0, minValidationText + '0')
+    .max(5, maxValidationText + '5')
+    .required(),
+  s3ComponentsScore: Yup.number()
+    .min(0, minValidationText + '0')
+    .max(7, maxValidationText + '7')
+    .required(),
+  s4DescribePotentialEconomicBenefits: Yup.number()
+    .min(0, minValidationText + '0')
+    .max(1, maxValidationText + '1')
+    .required(),
+  s5DetailsHowATNPWillAddressSafetyConcerns: Yup.number()
+    .min(0, minValidationText + '0')
+    .max(1, maxValidationText + '1')
+    .required(),
+  s6DescribeConsultationUndertaking: Yup.number()
+    .min(0, minValidationText + '0')
+    .max(1, maxValidationText + '1')
+    .required(),
```

Comment:
I'm assuming all of them are required. Maybe it's worth still confirming, though?